configs/example-srk.yaml

```diff
@@ -34,4 +36,51 @@ service :
       # Optional vpc/security-group setup to use.
       # e.g.: "vpc-123456789abcdef,sg-123456789abcdef"
       vpc-config : null
+      # Optional custom runtime and layer configuration
+      runtimes :
+        # example custom runtime definition
+        cffs-python :
+          # AWS runtime as base, use 'provided' for none
+          base : provided
+          # list of additional layers
+          layers :
+            - 'arn:aws:lambda:eu-central-1:669282166790:layer:runtime-python37:3'
+      # Optional default runtime if runtime is not provided by CLI
+      default-runtime : 'python3.7'
```

Comment:
In general you can specify a "base" to your "runtime" which then refers to FaaS provided runtimes like python3.7. In this specific case that does not work because cffs-python already includes the Python runtime.